### PR TITLE
Corrected the workflow import 

### PR DESCRIPTION
**Pull request by areksolek**
Corrected the workflow import

### CODE REVIEW

File: modules/Settings/Workflows/models/Module.php [hunk 1]
```diff
@@ -252,15 +252,16 @@ public static function exportTaskMethod($methodName)
...
-			if (file_put_contents($method['function_path'], $scriptData) === false) {
-				$messages['error'][] = \App\Language::translate('LBL_FAILED_TO_SAVE_SCRIPT', $this->getName(true), basename($method['function_path']), $method['function_path']);
+			if (file_put_contents($method['function_path'], $scriptData) == false) {
+				$messages['error'][] = \App\Language::translate('LBL_FAILED_TO_SAVE_SCRIPT', $moduleName) . ' ' . basename($method['function_path']) . ' ' . \App\Language::translate('LBL_TO', $moduleName) . ' ' . $method['function_path'];
 			}
 		} else {
 			require_once $method['function_path'];
 			if (!function_exists($method['function_name'])) {
-				$messages['error'][] = \App\Language::translate('LBL_SCRIPT_EXISTS_FUNCTION_NOT', $this->getName(true), $method['function_name'], $method['function_path']);
+				$messages['error'][] = \App\Language::translate('LBL_SCRIPT_EXISTS_FUNCTION_NOT', $moduleName) . ' ' . $method['function_name'] . ' ' . \App\Language::translate('LBL_TO', $moduleName) . ' ' . $method['function_path'];
```
Comment (TomaszQr) [inline modules/Settings/Workflows/models/Module.php]: ?

File: modules/Settings/Workflows/models/Module.php [hunk 2]
```diff
@@ -252,15 +252,16 @@ public static function exportTaskMethod($methodName)
 	 */
 	public function importTaskMethod(array &$method, array &$messages)
 	{
+		$moduleName = $this->getName(true);
 		if (!file_exists($method['function_path'])) {
 			$scriptData = base64_decode($method['script_content']);
-			if (file_put_contents($method['function_path'], $scriptData) === false) {
-				$messages['error'][] = \App\Language::translate('LBL_FAILED_TO_SAVE_SCRIPT', $this->getName(true), basename($method['function_path']), $method['function_path']);
+			if (file_put_contents($method['function_path'], $scriptData) == false) {
+				$messages['error'][] = \App\Language::translate('LBL_FAILED_TO_SAVE_SCRIPT', $moduleName) . ' ' . basename($method['function_path']) . ' ' . \App\Language::translate('LBL_TO', $moduleName) . ' ' . $method['function_path'];
```
Comment (TomaszQr) [inline modules/Settings/Workflows/models/Module.php]: ?

File: languages/en_us/Settings/Workflows.json [hunk 3]
```diff
@@ -57,6 +57,7 @@
     "LBL_THE_SAME_VALUE_IS_USED_FOR_START_DATE": "The same value is used for the start date",
     "LBL_TIME": "Time",
     "LBL_TITLE": "Title",
+    "LBL_TO": "to",
```
Comment (TomaszQr) [inline languages/en_us/Settings/Workflows.json]: ?

File: modules/Settings/Workflows/models/Record.php [hunk 4]
```diff
@@ -325,8 +325,9 @@ public function executionConditionAsLabel($executionCondition = null)
 			$executionCondition = $this->get('execution_condition');
 		}
 		$arr = ['ON_FIRST_SAVE', 'ONCE', 'ON_EVERY_SAVE', 'ON_MODIFY', 'ON_DELETE', 'ON_SCHEDULE', 'MANUAL', 'TRIGGER', 'BLOCK_EDIT', 'ON_RELATED'];
-
-		return $arr[$executionCondition - 1];
+		if (!empty($arr[$executionCondition - 1])) {
+			return $arr[$executionCondition - 1];
```
Comment (rskrzypczak) [inline modules/Settings/Workflows/models/Record.php]: return  isset($arr[$executionCondition - 1]) ?  $arr[$executionCondition - 1] : '';
no skip return

**Pull request merged**